prow/scripts/cluster-integration/serverless-integration-k3s.sh

```diff
@@ -85,6 +88,45 @@ docker run -d \
   eu.gcr.io/kyma-project/test-infra/docker-registry-2:20200202
 }
 
+install::kyma_cli() {
```

Comment:
Can you use existing function for installing kyma_cli? See https://github.com/kyma-project/test-infra/blob/c4d392462597523d632df0c42f8332d0ae52f3d8/prow/scripts/lib/kyma.sh#L114

Reply:
Unfortunately, no. This script is [sent over](https://github.com/kyma-project/test-infra/pull/3656/files#diff-7588c996360f463f2a29b2718d12f8986ed77df00fbf089c6a5c71adb805fc64R117) to bash over gcloud-ssh. It has to be self-contained and can't source any external files.